openstates/ny/utils.py

```diff
@@ -45,7 +45,7 @@ def __repr__(self):
     @CachedAttr
     def text(self):
         text = self.scraper.get(self.url).text
-        self.urls_object.validate(self.name, self.url, text)
```

Comment:
what error was this raising?

Reply:
File "/media/bikram/0bb967d3-86e9-4eee-934c-c0fc3f76980d/home/bikram/openstates/openstates/ny/utils.py", line 148, in validate
    for validator in self._validators[name]:
AttributeError: 'Urls' object has no attribute '_validators'